configs/gtp-raw.cfg

```diff
@@ -0,0 +1,5 @@
+# GTP config for KataGo-raw
+@include gtp-base.cfg
+
+maxVisits = 32
+numSearchThreads = 6    # Default from gtp_example.cfg
```

Comment:
Should we set this to 1 by default to get more stable results?
The multithreaded search logic is kind of complicated and non-deterministic and I'm worried it might interact badly with low visit counts.

Reply:
It seems like this shouldn't be a problem with a sufficiently large sample size— we'll just be Monte Carlo integrating over the different nondeterministic code paths.

Reply:
Mm, the main argument I have in favor of numSearchThreads is that we use it everywhere else so this us maintain consistent results. e.g. we may want to compare the strength of KataGo to ELF at various visits, and it'd be nice if we didn't have to worry about numSearchThreads confounding our results slightly.

Reply:
Our runs involving GTP are pretty cheap as overall % of compute so I lean towards setting `numSearchThreads=1` and eating the performance cost. We can also tweak it later if it ends up being annoyingly slow (and do an ablation to check it doesn't matter).